src/applications/mhv/medical-records/actions/careSummariesAndNotes.js

```diff
@@ -12,10 +12,29 @@ export const getCareSummariesAndNotesList = () => async dispatch => {
   }
 };
 
-export const getCareSummaryAndNotesDetails = noteId => async dispatch => {
+export const getCareSummaryAndNotesDetails = (
+  noteId,
+  noteList,
+) => async dispatch => {
   try {
-    const response = await getNote(noteId);
-    dispatch({ type: Actions.CareSummariesAndNotes.GET, response });
+    // Check if noteList has data
+    if (noteList && noteList.length > 0) {
```

Comment:
These check methods all seem very similar - would it be possible to use a single util to handle the logic of whether to call the backend or not for all domains?